src/routes/Private.jsx

```diff
@@ -115,13 +82,10 @@ class Private extends React.Component {
                   <div className={classes.component}>
                     <Switch>
                       <Route exact path="/" component={HomeView} />
-                      {features.searchui && (
-                        <Route exact path="/explorer" component={DataExplorerView} />
-                      )}
-                      <Route exact path="/datasets" component={DatasetsView} />
+                      <Route exact path="/datasets" component={HomeView} />
+                      <Route exact path="/snapshots" component={HomeView} />
                       <Route exact path="/datasets/:uuid/query" component={QueryView} />
```

Comment:
presumably, we'll want these paths to live under `HomeView` too

Reply:
I don't think we need to since these pages aren't under the datasets/snapshots tab in the new mocks